### ISSUE TITLE
Expandable on Row Click

### ISSUE THREAD

**Issue by emin-karadag** [jbetancur/react-data-table-component#545]
Hello there,
Let me first mention the versions I use.

**React**: ^16.13.1
**react-data-table-component**: ^6.7.1

My problem is with expandable lines.
If there is only text in a cell and you click on it, row expansion works. However, the row expansion does not work when there is a picture or an ordinary div and not a text.

When I click on the area marked in red in the image below, row expansion does not work. However, it works when I click on the area I marked in blue.

![bug](https://user-images.githubusercontent.com/7556164/79859597-ea48c180-83d9-11ea-98f9-d884702e7823.PNG)

The codes I use for this process are as follows.

`import React, { Component } from 'react';
import DataTable from 'react-data-table-component';

const data = [{ id: 1, title: 'Conan the Barbarian', year: '1982' }];
const columns = [
  {
    name: 'Title',
    selector: 'title',
    sortable: true,
    cell: row => <div>{row.title}</div>
  },
  {
    name: 'Year',
    selector: 'year',
    sortable: true,
    right: true,
  },
];

const ExpanableComponent = ({ data }) => <div>{data.title}</div>;

class App extends Component {
  render() {
    return (
      <div>
        <DataTable
          title="Arnold Movies"
          columns={columns}
          data={data}
          expandOnRowClicked
          expandableRows
          expandableRowsComponent={<ExpanableComponent />} />
      </div>
    )
  }
}

export default App;`

**Comment by jbetancur** [comment]
This seems to happen with any custom cell. The reason is that there is a wrapper div that is used for native cells to propagate events. But when using custom cells is where it is not working.

As a workaround, you can try adding the following to your custom cell component `data-tage="___react-data-table-allow-propagation___"` and see if that works.

I'll see about getting a long term fix in for this.

**Comment by jbetancur** [comment]
See https://github.com/jbetancur/react-data-table-component#61-columns and the `cell` section. 

## TL;DR
if you are using properties such as: onRowClicked, onRowDoubleClicked, expandOnRowClicked or expandOnRowDoubleClicked then those click events will be ignored when clicking on your custom cell. To allow RowClicked events you can add data-tag="allowRowEvents" to your custom cell component elements.
